python/dgl/sparse/elementwise_op.py

```diff
@@ -16,7 +16,7 @@ def add(
     The supported combinations are shown as follows.
 
     +--------------+------------+--------------+--------+
-    |     A \ B    | DiagMatrix | SparseMatrix | scalar |
+    |    A \\ B    | DiagMatrix | SparseMatrix | scalar |
     +--------------+------------+--------------+--------+
     |  DiagMatrix  |     ✅     |      ✅      |   🚫   |
     +--------------+------------+--------------+--------+
```

Comment:
Fix the position of "|" for L21.

Reply:
It is actually correct, the emoji took 2 char space.